plugins/catalog-backend/src/ingestion/CatalogRules.ts

```diff
@@ -76,6 +78,14 @@ export class DefaultCatalogRulesEnforcer implements CatalogRulesEnforcer {
    * catalog:
    *   rules:
    *   - allow: [Component, API]
+   *   - allow: [Template]
+   *     locations:
+   *       - type: url
+   *         pattern: https://github.com/org/*\/blob/master/template.yaml
+   *   - allow: [Location]
+   *     locations:
+   *       - type: url
+   *         pattern: https://github.com/org/repo/blob/master/location.yaml
```

Comment:
What happens if two of these rules apply? Are they applied in any order? Do you apply the first match top to bottom (start with index 0)? (This would feel natural to stop at the first match, however these docs would not work like that as the first one always matches.)